eta/core/vgg16.py

```diff
@@ -44,28 +44,19 @@
 import tensorflow as tf
 
 from eta.core.config import Config
-from eta import constants
+import eta.core.image as etai
 from eta.core.features import Featurizer
-import eta.core.image as im
-from eta.core.weights import Weights, WeightsConfig
+import eta.core.models as etam
 
 
 logger = logging.getLogger(__name__)
 
 
-DEFAULT_CONFIG_PATH = os.path.join(constants.CONFIGS_DIR, 'vgg16-config.json')
-
-
 class VGG16Config(Config):
     '''Configuration settings for the VGG16 network.'''
 
     def __init__(self, d):
-        self.weights = self.parse_object(d, "weights", WeightsConfig)
-
-    @classmethod
-    def load_default(cls):
-        '''Loads the default config file.'''
-        return cls.from_json(DEFAULT_CONFIG_PATH)
+        self.model = self.parse_string(d, "model", default="VGG-16")
```

Comment:
Heavens why add the dash when everywhere else it is VGG16!!

Reply:
We can update when a convention arises